README.md

```diff
@@ -1 +1,40 @@
-# LeaderBoard-project
\ No newline at end of file
+# LeaderBoard-project
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This is the simple to do list: using webpack that shows of my accomplishments as a software developer.In this project  we learn how to setup the webpack and nodemodules. This project do the to-do-list add , remove and edit function with local storage.
+
+
+
+## Built With
+
+- HTML
+- Css
+- Javascript
+- Es6
+- webpack
+- gitflow
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## live version
+
+### Prerequisites
+- A simple web browser (preferably Chrome or Firefox)
+- An IDE
+
+### Setup
+- Download this repository and open index.html file.
+
+## Authors
+
+👤 **Soe Sandar Win**
+
+- GitHub : [@soesandarwin2201] (https://github.com/soesandarwin2201)
+- LinkedIn : [soesandarwin] (https://www.linkedin.com/in/soe-sandar-win-softwareengineer/)
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
```

Comment:
- [ ] Optional:
- Please add the Licence in your readme. `MIT` will be fine. It makes people trust your project. Also professional.